### DISCUSSION TITLE
Natives using gamerHandle parameter!

### DISCUSSION THREAD

**Discussion by aoqia194** [YimMenu/YimMenu#382] [Q&A] [answered]
I am just wondering if anyone knows a way to get the gamerHandle from a player.
Also, in which class does the player's gamerHandle reside?
And yes, I've used google and searched on the UnknownCheats forum for information about this and I couldn't find relevant information to tell me where I can find it or explaining it.

**Comment by maybegreat48** [comment, marked as answer]
There are two gamer handles, one is the `rage::rlGamerHandle` which you can find in `netPlayerData::m_rockstar_id2` and the other one is the serialized version of the former handle that you can use with natives some pseudocode for that:
```c++
uint64_t handle[13];
NETWORK::NETWORK_HANDLE_FROM_PLAYER(player, (Any*)handle, 13);
NETWORK::NETWORK_SEND_TEXT_MESSAGE("test", (Any*)handle);
```

**Reply by aoqia194** [reply]
Thank you so much :D
